### PR TITLE
feat: upload CV

### PR DESCRIPTION
**Pull request by LiKa12345**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+</p>
+
+<p>
+  <strong> April 2019-Present </strong>  Customer Support Specialist,  Evolution Gaming LLC 
+</p>
+<p>
+    <strong >October 2016-March2019 </strong>   Network Troubleshooter, JCS Silknet
+</p>
+<p>
+    <strong>December2008-October201 </strong>6 Telephone Station Manager, Akhali Kselebi Ltd
```
Comment (Xelsoid) [inline index.html:57]: image

Comment (Xelsoid) [reply index.html:57]: Please correct the indents in your code

File: style.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    
+}
+
+.h {
```
Comment (Xelsoid) [inline style.css:8]: It's better to name classes with sense (.header) and so on. Do not use shorthand.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
```
Comment (Xelsoid) [inline index.html:1]: Please correct all indents in the code

File: index.html (hunk 1 above)
Comment (LiKa12345) [reply index.html]: Thank you. It's already fixed.

File: style.css (hunk 2 above)
Comment (LiKa12345) [reply style.css:8]: ok, I will take into consideration.

File: index.html (hunk 3 above)
Comment (LiKa12345) [reply index.html:1]: Done!

File: style.css (hunk 2 above)
Comment (Xelsoid) [reply style.css:8]: please correct this gap

Comment (LiKa12345) [reply style.css]: Done.

**Comment by Xelsoid** [comment]
Fine now. Approved. You can merge this branch.

**Pull request merged**